### ISSUE TITLE
lando-messaging pickling issue

### ISSUE THREAD

**Issue by dleehr** [Duke-GCB/lando-messaging#20]
Moved from https://github.com/Duke-GCB/bespin/issues/6

> During the upgrade to python3 we found that messages sent from bespin-api(python3) could not be read by lando(python2) and images containing lando. There is an option to specify the protocol in the `dumps` command: https://docs.python.org/3/library/pickle.html#pickle.dumps
To try and keep the older images compatible we can use this option.
> 
> We need to 
> - change [lando-messaging](https://github.com/Duke-GCB/lando-messaging) to set the protocol to 2
> - update lando-messaging in bespin-api, bespin-mailer, and lando
> - update lando to python3

**Comment by dleehr** [comment]
Note that we have a lando messaging version check, so even though we're now pickling with a protocol compatible with python2, older versions of lando-messaging will raise if the versions don't match:

```
File "/usr/local/lib/python3.6/site-packages/lando_messaging-0.7.4-py3.6.egg/lando_messaging/workqueue.py", line 181, in raise_on_version_mismatch
   raise ValueError("Received version mismatch.")
```

**Comment by dleehr** [comment]
Fixed in #19, #22, and PRs for other projects.
